docs/getting-started/agentless-troubleshooting.asciidoc

```diff
@@ -0,0 +1,42 @@
+[[agentless-integration-troubleshooting]]
+= Agentless integrations FAQ
+
+Frequently asked questions and troubleshooting steps for {elastic-sec}'s agentless CSPM integration.
+
+*When I make a new integration, how long until the agent appears on the Integration Policies page?*
+
+After you create a new agentless integration, the new integration policy may show a button that says **Add agent** instead of the associated agent for several minutes during agent enrollment. No action is needed other than refreshing the page once enrollment is complete.
+
+*How do I troubleshoot an "Offline" agent?*
+
+For agentless integrations to successfully connect to {elastic-sec}, your Fleet server host value must be the default. Otherwise, the agent status that appears on the Fleet page will be `Offline`, and logs will include the following error `[elastic_agent][error] Cannot checkin in with fleet-server, retrying`. 
+
+To troubleshoot this issue:
+
+. Find **Fleet** in the navigation menu or use the {kibana-ref}/introduction.html#kibana-navigation-search[global search field]. Go to the **Settings** tab.
+. Under **Fleet server hosts**, click the **Actions** button for the policy named `Default`. This opens the **Edit Fleet Server** flyout. The policy named `Default` should have the **Make this Fleet server the default one** setting enabled. If not, enable it, then delete your integration and create it again.
+. If the **Make this Fleet server the default one** setting was already enabled, it's possible someone changed the **URL** value for your default fleet server. In this case, contact Elastic support to find out what the original **URL** value was, update the settings to match this value, then delete your integration and create it again.
```

Comment:
This looks related to the second part of the previous step but also seems like a duplicate step. Is this step an extension of step 2?